detection-rules/malicious_slack_html.yml

```diff
@@ -0,0 +1,42 @@
+name: "Malicious Slack download attachment"
+description: |
+  Impersonation of the video conferencing provider Slack, encouraging the user to download a fake Slack client.
+references:
+  - "https://delivr.to/payloads?id=790c79d0-3ecc-4bcf-aa51-f417c65c5983"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound 
+   and sender.email.domain.root_domain not in ("slack.com")
+   and (
+       body.html.display_text != ""
+       and strings.ilike(body.html.display_text, "*Slack*for*Windows*")
+       and strings.ilike(body.html.display_text, "*download*")
+     or any(attachments,
+           (
+               .file_extension in~ ("html", "htm", "shtml", "dhtml") or
+               .file_extension in~ $file_extensions_common_archives or
+               .file_type == "html"
+           )
+           and any(beta.binexplode(.),
+                   strings.ilike(.scan.html.title, "*slack*") and
+                   any(.scan.url.urls, strings.ilike(.path, '*exe*', '*msi*', '*zip*'))
+                 )
+     )
+    or any(body.links,
+    // Slack branding
+    beta.linkanalysis(.).credphish.brand.name == "Slack" and 
+    beta.linkanalysis(.).credphish.brand.confidence == "high" and 
+    beta.linkanalysis(.).credphish.disposition == "phishing" and 
+
+    any(beta.binexplode(beta.linkanalysis(.).screenshot),
+        all([
+            "download",
+            "app",
+            "version"
+        ], strings.icontains(..scan.ocr.raw, .))
+    )
+    )
+   )
+tags:
```

Comment:
```suggestion
name: "Malicious Slack download attachment"
description: |
  Impersonation of the video conferencing provider Slack, encouraging the user to download a fake Slack client.
type: "rule"
references:
  - "https://delivr.to/payloads?id=790c79d0-3ecc-4bcf-aa51-f417c65c5983"
severity: "high"
source: |
  type.inbound 
  and sender.email.domain.root_domain not in ("slack.com")
  and (
    (
      body.html.display_text != ""
      and strings.ilike(body.html.display_text, "*Slack*for*Windows*")
      and strings.ilike(body.html.display_text, "*download*")
    )
    or (
      any(attachments,
        (
          .file_extension in~ ("html", "htm", "shtml", "dhtml") or
          .file_extension in~ $file_extensions_common_archives or
          .file_type == "html"
        )
        and any(beta.binexplode(.),
          strings.ilike(.scan.html.title, "*slack*") and
          any(.scan.url.urls, strings.ilike(.path, '*exe*', '*msi*', '*zip*'))
        )
      )
    )
    and (
      any(body.links,
        beta.linkanalysis(.).credphish.brand.name == "Slack" and 
        beta.linkanalysis(.).credphish.brand.confidence == "high" and 
        beta.linkanalysis(.).credphish.disposition == "phishing" and 
        any(beta.binexplode(beta.linkanalysis(.).screenshot),
          all([
            "download",
            "app",
            "version"
          ], strings.icontains(..scan.ocr.raw, .))
        )
      )
    )
  )
tags:
  - "Brand impersonation"
```
@aidenmitchell  What do you think about moving that second OR to an AND. This way we can strengthen the body text anchor to have an AND on the body.links section being true also?  I think this logically makes sense, please sanity check me :)

Reply:
This isn't logically operating the way I want it to in testing. i'll ping you in slack about my thoughts for next steps.